client/app/scripts/components/footer.js

```diff
@@ -76,7 +79,7 @@ class Footer extends React.Component {
             title={forceRelayoutTitle}>
             <span className="fa fa-refresh" />
           </a>
-          <a className="footer-icon" href={otherContrastModeUrl} title={otherContrastModeTitle}>
+          <a onClick={this.handleContrastClick} className="footer-icon" href="" title={otherContrastModeTitle}>
```

Comment:
Is it necessary to put an explicit `href=""` here? Just looking at the layout refresh `<a>` above which doesn't have it..